### ISSUE TITLE
Blender - specify that Cycles GPU rendering is not supported

### ISSUE THREAD

**Issue by pawelostr** [ThatGuySam/doesitarm#377]
I think it would make a difference for some, as so far you can't render using GPU, and as fast as the M1 is, it's still far from any GPU render. I propose adding something along the lines of "Cycles rendering using GPU not supported".

<img width="772" alt="Zrzut ekranu 2020-12-2 o 16 46 39" src="https://user-images.githubusercontent.com/26488603/100895714-27bc1400-34be-11eb-821d-f76161917d5d.png">

**Comment by ThatGuySam** [comment]
@pawelostr Is this the only or the biggest asterisks kind of detail for Blender running on Rosetta 2?  

If so we can go ahead and add it to the end of the status. 

Otherwise, we can add it as a button below along with any other issues that come up.

**Comment by pawelostr** [comment]
@ThatGuySam I would say it's an only, but a big issue. Basically everything works on the software side and I would say Rosetta isn't to blame, it's just that OpenCL nor CUDA is supported from what I understand. It's working fine, except that the performance is far off for any kind of serious work (when rendering final work or when one wants accurate live preview, if I had to put a number on it - 5x) and no-one should currently buy M1 if they're serious about working in Blender. 

tl:dr At the end of the status

**Comment by ThatGuySam** [comment]
Sounds good! 

Do you recommend any Youtube videos or anything to give people a more detailed understanding of how performance is affected?

**Comment by pawelostr** [comment]
https://blog.thepixelary.com/post/160210224042/cpu-vs-gpu-rendering-in-blender-cycles

But I'm pretty sure that people affected by it will know what's up, as it's kinda like trying to play a modern game without a dedicated GPU (or even the CPU itself)
